backend/tests/test_server.py

```diff
@@ -687,3 +686,158 @@ def test_project_translations_file_path_traversal(
 
     # Should give error status
     assert response.status_code == 422
+
+
+def test_project_settings_with_chat_profile_config_overrides(
+    test_client: TestClient,
+    test_config: ChainlitConfig,
+    monkeypatch: pytest.MonkeyPatch,
+):
+    """Test that /project/settings endpoint returns merged configuration when chat_profile is specified."""
+    from chainlit.config import (
+        ChainlitConfigOverrides,
+        FeaturesSettings,
+        McpFeature,
+        UISettings,
+    )
+    from chainlit.types import ChatProfile
+
+    # Mock chat profiles with different config overrides
+    mock_profiles = [
+        ChatProfile(
+            name="basic",
+            markdown_description="Basic profile without overrides",
```

Comment:
This markdown description does not take into account the "language" param right? Since this /project/settings endpoint will return the description as is, and the application does not know which language user is using until user send a GET request to /project/settings

Reply:
Yeah that is true.